Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCardModel.swift

```diff
@@ -12,6 +12,7 @@ public struct PPOProjectCardModel: Identifiable, Equatable {
   public let address: String?
   public let actions: (Action, Action?)
   public let tierType: TierType
+  public let backingDetailsUrl: String
```

Comment:
Would be nice if we could use a stronger type here, but I see that this ultimately ends up in SurveyResponseViewController, which is configured with a String, so that's fine.